### ISSUE TITLE
[BUG] Overriding DAB lookup variables in targets breaks variables view

### ISSUE THREAD

**Issue by jamuska** [databricks/databricks-vscode#1410]
**Describe the bug**
When you define a lookup variable in `databricks.yml` and override it with another value under some target, `Bundle variables` view does not show variable values.

**To Reproduce**
1. Init new bundle `databricks bundle init default-python`
2. Define variables in `databricks.yml`
```
variables: 
  test: test_value
  lookup_test:
    lookup: 
      service_principal: test_service_principal
```
3. Override `lookup_test` under `dev` target
```
targets:
  dev:
    variables:
      lookup_test:
        lookup:
          service_principal: override_service_principal
```
4. `Bundle variables` view becomes empty

**Screenshots**
Without overridden lookup variable
![image](https://github.com/user-attachments/assets/6fe46025-37fe-4506-87c2-267d37d35265)

After overriding lookup variable
![image](https://github.com/user-attachments/assets/e1ce8681-d8fd-47a3-9b69-575a4bf29978)

**System information:**

1. Paste the output ot the `Help: About` command (CMD-Shift-P).
Version: 1.94.2 (user setup)
Commit: 384ff7382de624fb94dbaf6da11977bba1ecd427
Date: 2024-10-09T16:08:44.566Z
Electron: 30.5.1
ElectronBuildId: 10262041
Chromium: 124.0.6367.243
Node.js: 20.16.0
V8: 12.4.254.20-electron.0
OS: Windows_NT x64 10.0.22631
3. Databricks Extension Version
2.4.7
**Databricks Extension Logs**
Please attach the [databricks extension logs](https://docs.databricks.com/dev-tools/vscode-ext.html#send-usage-logs-to-databricks)
I cannot get Open Full logs from command palette to work when running vscode with WSL remote. I will try to add these later.

**Additional context**

**Comment by ilia-db** [comment]
Thanks for reporting it, the fix should be released this week

**Comment by jamuska** [comment]
> Thanks for reporting it, the fix should be released this week

Great to hear, many thanks for a quick response.

**Comment by ilia-db** [comment]
The fix has been released in 2.4.8
